src/quark-core/src/QuarkWallet.sol

```diff
@@ -219,7 +222,9 @@ contract QuarkWallet is IERC1271 {
 
         emit ExecuteQuarkScript(msg.sender, op.scriptAddress, op.nonce, ExecutionType.Signature);
```

Comment:
Not a huge deal, but I wonder if this and the next line should be reversed? Probably has ~ no effect